modern/src/MainController.js

```diff
@@ -0,0 +1,22 @@
+import {  useDispatch, useSelector } from 'react-redux';
+import { connect } from 'react-redux';
+import { geofencesActions } from './store';
+import { useEffectAsync } from './reactHelper';
+
+
+const MainController = () => {
```

Comment:
Where do we load devices? Maybe we can load geofences there as well.

Reply:
Yes we can but my thought process was to keep the SocketController separate from rest of the things like Geofences, Groups, Maintenance etc. If you would like to keep all the fetches in  SocketController only then I'll move it there.

Reply:
Maybe we need a better name then. Something like caching controller?

Reply:
CachingController/ StoreController